kolibri/plugins/device/assets/src/app.js

```diff
@@ -21,7 +21,8 @@ class DeviceManagementModule extends KolibriApp {
     const isLearnOnlyDevice = plugin_data.isSubsetOfUsersDevice;
     const isSuperuser = store.getters.isSuperuser;
     const isFacilityAdmin = store.getters.isFacilityAdmin;
-    if (isLearnOnlyDevice && isSuperuser && !isFacilityAdmin) {
+    const userCanManageContent = store.getters.getUserPermissions['can_manage_content'];
```

Comment:
This is more an indication of the lack of consistency in the session getters than your code here, but there is also a direct getter for this too: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/state/modules/session.js#L38

Reply:
Thanks @rtibbles for the heads up. I will make this minor fix in the https://github.com/learningequality/kolibri/pull/10190 pr.